File: server/iih_pub/iih.ci_pub/src/main/java/iih/ci/ord/ciordems/d/EmsFeesDTO.java
package iih.ci.ord.ciordems.d;

import xap.mw.core.data.*;
import xap.mw.coreitf.d.*;
import java.math.BigDecimal;

/**
 * 实体 DTO数据 
 * 
 */
public class EmsFeesDTO extends BaseDTO {
	private static final long serialVersionUID = 1L;
	
	/**
	 * 服务类别名称
	 * @return String
	 */
	public String getName() {
		return ((String) getAttrVal("Name"));
	}
	/**
	 * 服务类别名称
	 * @param Name
	 */
	public void setName(String Name) {
		setAttrVal("Name", Name);
	}
	/**
	 * 金额合计
	 * @return FDouble
	 */
	public FDouble getAllfees() {
		return ((FDouble) getAttrVal("Allfees"));
	}
	/**
	 * 金额合计
	 * @param Allfees
	 */
	public void setAllfees(FDouble Allfees) {
		setAttrVal("Allfees", Allfees);
	}
	/**
	 * 已缴费金额
	 * @return FDouble
	 */
	public FDouble getHaspay() {
		return ((FDouble) getAttrVal("Haspay"));
	}
	/**
	 * 已缴费金额
	 * @param Haspay
	 */
	public void setHaspay(FDouble Haspay) {
		setAttrVal("Haspay", Haspay);
	}
	/**
	 * 未缴费
	 * @return FDouble
	 */
	public FDouble getNopay() {
		return ((FDouble) getAttrVal("Nopay"));
	}
	/**
	 * 未缴费
	 * @param Nopay
	 */
	public void setNopay(FDouble Nopay) {
		setAttrVal("Nopay", Nopay);
	}
}

File: client/iih_stub/iih.ci_stub/iih.ci.ord/ciordems/d/EmsFeesDTO.cs
using System;
using xap.mw.core.data;
using xap.mw.coreitf.d;

namespace iih.ci.ord.ciordems.d
{
    /// <summary>
    /// EmsFeesDTO 的摘要说明。
    /// </summary>
    public class EmsFeesDTO : BaseDTO {

        public EmsFeesDTO() {
 
        }

        /// <summary>
        /// 服务类别名称
        /// </summary>
		public string Name {
            get { return getAttrVal<string>("Name",null); }
            set { setAttrVal<string>("Name", value); }
        }

        /// <summary>
        /// 金额合计
        /// </summary>
		public FDouble Allfees {
            get { return getAttrVal<FDouble>("Allfees",null); }
            set { setAttrVal<FDouble>("Allfees", value); }
        }

        /// <summary>
        /// 已缴费金额
        /// </summary>
		public FDouble Haspay {
            get { return getAttrVal<FDouble>("Haspay",null); }
            set { setAttrVal<FDouble>("Haspay", value); }
        }

        /// <summary>
        /// 未缴费
        /// </summary>
		public FDouble Nopay {
            get { return getAttrVal<FDouble>("Nopay",null); }
            set { setAttrVal<FDouble>("Nopay", value); }
        }
        /// <summary>
        /// 返回属性列表
        /// </summary>
        /// <returns></returns>
        public override string[] getAttrNames()
        {
            return new string[]{ "Name", "Allfees", "Haspay", "Nopay"};
        }
        
        /// <summary>
        /// 返回全路径DO类名
        /// </summary>
        /// <returns></returns>
        public override string getFullClassName()
        {
            return "iih.ci.ord.ciordems.d.d.EmsFeesDTO";
        }
    }
}
